srcpkgs/qmidiarp/patches/cross.patch

```diff
@@ -0,0 +1,15 @@
+--- configure.ac
++++ configure.ac
+@@ -127,12 +127,6 @@ if test "x$ac_buildapp" = "xyes" -o ["x$ac_lv2pluginuis" = "xyes"  -a "x$ac_lv2w
+           QT_INCLUDE_DIR=`$PKG_CONFIG --variable=includedir Qt5Gui`
+           QT_CXXFLAGS=`$PKG_CONFIG --cflags Qt5Core Qt5Gui Qt5Widgets`
+           QT_CXXFLAGS="$QT_CXXFLAGS -fPIC"
+-          EXTRA_QT_INCLUDE_DIR="$QT_INCLUDE_DIR/Qt"
+-          AC_CHECK_FILE([$QT_INCLUDE_DIR/QtWidgets/QWidget],
+-            AC_MSG_NOTICE([No extra QT_INCLUDE_DIR needed]),
+-              AC_CHECK_FILE([$EXTRA_QT_INCLUDE_DIR/QtWidgets/QWidget],
+-                QT_CPPFLAGS="$QT_CPPFLAGS -I$EXTRA_QT_INCLUDE_DIR",
+-                  AC_MSG_WARN([QWidget not found])))
```

Comment:
Is the removal because of the `-I$EXTRA_QT_INCLUDE_DIR` inclusion?

Reply:
This removes a cross-unaware file existence check that breaks the build:
```
checking for /usr/include/qt5/QtWidgets/QWidget... configure: error: cannot check for file existence when cross compiling
=> ERROR: qmidiarp-0.6.5_1: do_configure: '${configure_script} ${configure_args}' exited with 1
=> ERROR:   in do_configure() at common/build-style/gnu-configure.sh:8
```
It's unnecessary too, because the `pkg-config` based check succeeds and returns the same include paths:
```
checking for Qt5Core Qt5Gui Qt5Widgets >= 5.0... yes
```